packages/design-system/vue/cypress/support/component-index.html

```diff
@@ -1,11 +1,11 @@
 <!DOCTYPE html>
-<html>
+<html class="dark">
 <head>
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width,initial-scale=1.0">
   <title>Components App</title>
-
+  <link rel="stylesheet" href="/__cypress/src/src/assets/style.css">
```

Comment:
Why is this necessary when we already do this here? https://github.com/eternagame/eterna-next/blob/main/packages/design-system/vue/cypress/support/component.ts#L23

Reply:
Now I'm not sure why I added this, but you're right it's unnecessary.